oz/redis_sessions/middleware.py

```diff
@@ -24,7 +24,17 @@ def _session_key(self):
 
             if not session_id:
                 session_id = oz.redis_sessions.random_hex(20)
-                self.set_secure_cookie("session_id", session_id.encode('utf-8'), httponly=True)
+                if "cookie_domain" in oz.settings:
```

Comment:
It'd be more terse to just do this:

    self.set_secure_cookie(
        name="session_id",
        value=session_id.encode('utf-8'),
        domain=oz.settings.get("cookie_domain"),
        httponly=True)

It's also marginally faster because, if a cookie_domain is set, it involves just one lookup to the dictionary rather than two.